### ISSUE TITLE
Build issue with kernel 5.4

### ISSUE THREAD

**Issue by Mantra84** [oracle/ktf#25]
Hello,

While compiling kernel 5.4.125 with the latest KTF version (SHA-1 63c19dead80de9cd654b08120d28a04f24174f4b), I got the `nla_strscpy implicit declaration` error.

This issue should have been solved with the lines in ktf_compat.h (line 56):

> #if (KERNEL_VERSION(5, 11, 0) > LINUX_VERSION_CODE)
> #define nla_strscpy nla_strlcpy
> #endif

But it seems that this branch is not entered.

I think the issue come form the first condition in ktf_compat.h (line 1):

> #if (KERNEL_VERSION(5, 2, 0) > LINUX_VERSION_CODE) 

Maybe this line should be change to:

> --- a/kernel/ktf_compat.h
> +++ b/kernel/ktf_compat.h
> @@ -1,4 +1,4 @@
> -#if (KERNEL_VERSION(5, 2, 0) > LINUX_VERSION_CODE)
> +#if (KERNEL_VERSION(5, 11, 0) > LINUX_VERSION_CODE)

What do you think about it ?

Thank you.

**Comment by knuto** [comment]
You are quite right. I I think I'll propose we just remove the other define to avoid making this mistake again in the future as more compat stuff is needed.

**Comment by knuto** [comment]
I have updated knuto/ktf to fix the issue (pull request #26)

**Comment by Mantra84** [comment]
Thanks for the fix :smile:
